cpp/models/abm/parameters.h

```diff
@@ -316,45 +317,19 @@ struct TestParameters {
     UncertainValue specificity;
 };
 
-struct GenericTest {
-    using Type = TestParameters;
-    static Type get_default()
-    {
-        return Type{0.9, 0.99};
-    }
-    static std::string name()
-    {
-        return "GenericTest";
-    }
-};
-
-/**
- * @brief Reliability of an AntigenTest.
- */
-struct AntigenTest : public GenericTest {
-    using Type = TestParameters;
-    static Type get_default()
-    {
-        return Type{0.8, 0.88};
-    }
-    static std::string name()
-    {
-        return "AntigenTest";
-    }
-};
-
-/**
- * @brief Reliability of a PCRTest.
- */
-struct PCRTest : public GenericTest {
-    using Type = TestParameters;
-    static Type get_default()
+struct TestData {
```

Comment:
Please add a brief description to the parameter.

Reply:
It's now added.